src/Hazelcast.Net/Networking/NetworkingOptions.cs

```diff
@@ -91,6 +92,9 @@ private NetworkingOptions(NetworkingOptions other)
     /// <para>Note that it however does not guarantee that the operation will always be
     /// executed on the owner, as the member table is only updated every 10 seconds.</para>
     /// </remarks>
+    [Obsolete("This option has no effect, use RoutingMode instead.", false)]
+    [ExcludeFromCodeCoverage]
+    [EditorBrowsable(EditorBrowsableState.Never)]
     public bool SmartRouting { get; set; } = true;
```

Comment:
We need to support the "smart routing" config option for the foreseeable future as some of our customers will not be able to update their client configs easily and we don't want to introduce a breaking change.

In the Java client, we do this by overriding `setSmartRouting(boolean)` to set our `RoutingMode` based on the `boolean` provided.

Reply:
As I mentioned above, smart routing stays same by setting the `RoutingMode` as in Java client.